model/elements/src/main/kotlin/de/gleex/pltcmd/model/elements/combat/Weapon.kt

```diff
@@ -0,0 +1,8 @@
+package de.gleex.pltcmd.model.elements.combat
+
+/**
+ * Data that describes what a type of weapon is able to do.
```

Comment:
```suggestion
 * Data that describes what a weapon is able to do.
```
The class describes a _weapon_ not a _type of weapon_. And I think for our purposes that is perfectly fine for now.

With weapon _types_ you would need different weapons with that specific type which is one level of hierarchy too much, I believe.